tests/core/factory/test_validation_definition_factory.py

```diff
@@ -634,6 +639,105 @@ def _test_overwrite_existing_validation(self, context: AbstractDataContext):
         assert updated_vd.id == existing_vd.id
         context.validation_definitions.get(vd_name)
 
+    @pytest.mark.filesystem
+    @pytest.mark.xfail(
```

Comment:
Probably good to reference `GX-481` in the reason for xfailing these.

Reply:
[673a28e](https://github.com/great-expectations/great_expectations/pull/10960/commits/673a28e99c15f453ee82856d0cb16bd493ac9292)